analyzers/tests/SonarAnalyzer.UnitTest/TestCases/LinkedListPropertiesInsteadOfMethods.cs

```diff
@@ -7,10 +7,10 @@ class MyClass
 {
     void Basic(LinkedList<int> data)
     {
-        data.First(); // Noncompliant {{'First' property of 'LinkedList' should be used instead of the 'First()' extension method.}}
-//           ^^^^^
-        data.Last(); // Noncompliant {{'Last' property of 'LinkedList' should be used instead of the 'Last()' extension method.}}
-//           ^^^^
+        _ = data.First(); // Noncompliant {{'First' property of 'LinkedList' should be used instead of the 'First()' extension method.}}
```

Comment:
A lot of the calls (not all) have been changed to be assignments. Can I ask what's the reason for this change?

Reply:
So, in some cases the extension method was called without an assignment to improve the readability:
`data.First();`
The problem arises when we fix that issue with the LinkedList property:
`data.First.Value;`
This will raise a `CS0201: Only assignment, call, increment, decrement, await, and new object expressions can be used as a statement`. That's why I added the assignment with the discard variable to the invalid statements.